pom.xml

```diff
@@ -51,56 +51,12 @@
         <bundle.file.name>${bundle.build.dir}/${project.build.finalName}.jar</bundle.file.name>
         <min.port>37000</min.port>
         <max.port>37999</max.port>
+        <project.build.outputTimestamp>2021-04-12T10:00:00Z</project.build.outputTimestamp>
     </properties>
 
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>biz.aQute.bnd</groupId>
-                <artifactId>bnd-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>bnd-process</id>
-                        <goals>
-                            <goal>bnd-process</goal>
-                        </goals>
-                        <configuration>
-                            <bnd>
--noextraheader:true
--snapshot: SNAPSHOT
-Sling-Namespaces: slingevent=http://sling.apache.org/jcr/event/1.0
-Sling-Nodetypes: SLING-INF/nodetypes/event.cnd
-Import-Package: \
-    javax.servlet.http;resolution:=optional,\
-    org.apache.felix.inventory;resolution:=optional,\
-    javax.servlet;resolution:=optional,\
-    <!-- For build compatibility with Java 11 -->\
-    javax.annotation;version="[0.0,2)",\
-    *
-Export-Package: \
-    org.apache.sling.event.jobs,\
-    org.apache.sling.event.jobs.consumer,\
-    org.apache.sling.event.jobs.jmx
-
--conditionalpackage: \
-    org.apache.jackrabbit.util,\
-    org.apache.sling.commons.osgi,\
-    org.apache.sling.event.api
--includeresource: \
-    target/dependency/org.apache.sling.event.api.jar,\
-    @target/dependency/quartz.jar!/org/quartz/CronExpression.class,\
-    @target/dependency/quartz.jar!/org/quartz/ValueSet.class,\
-    @target/dependency/org.apache.sling.commons.osgi.jar!/org/apache/sling/commons/osgi/PropertiesUtil.class,\
-    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/ISO9075.class,\
-    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/ISO8601.class,\
-    @target/dependency/jackrabbit-jcr-commons.jar!/org/apache/jackrabbit/util/XMLChar.class
-Bundle-Classpath: ., org.apache.sling.event.api.jar
-                            </bnd>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
             <plugin>
                 <groupId>biz.aQute.bnd</groupId>
```

Comment:
This plugin is active by default: https://github.com/apache/sling-parent/blob/35ff6b88ff7e69a9da0f94be1997b7a72450282e/sling-bundle-parent/pom.xml#L166 no need to reference it in your pom.xml

Reply:
+1, done